requirements.txt

```diff
@@ -13,4 +14,4 @@ scan-build==2.0.13
 sentry-sdk
 observable
 pypng==0.0.20
-pyinstaller
\ No newline at end of file
+pyinstaller
```

Comment:
I forget, why was this added?

Reply:
Not sure why that shows up on the diff, I'm pretty sure that line is unchanged.